stdlib/iarrayLabels.mli

```diff
@@ -38,10 +38,11 @@ open! Stdlib
 type +'a t = 'a iarray
 (** An alias for the type of immutable arrays. *)
 
-external length : 'a iarray -> int = "%array_length"
+external length : local_ 'a iarray -> int = "%array_length"
```

Comment:
Some of the comments in this file have diverged from iarray.mli. They should probably be lined up.